### PR TITLE
Add Karn, Living Legacy emblem from Arena

### PR DESCRIPTION
**Pull request by SlightlyCircuitous**
Adds missing art for a token mentioned in issue https://github.com/Cockatrice/Magic-Token/issues/85#issuecomment-1269095070

I took screenshots at the highest graphics Arena would let me have and then did my best with the crop. Please let me know if I need to alter either image further.

### CODE REVIEW

**Comment by SlightlyCircuitous** [comment]
I just noticed that I have some of these images uploaded as jpegs and some as pngs; is there a perferred format?

**Comment by ebbit1q** [comment]
in my opinion using lossy formats like jpeg is fine, you don't really notice the difference in quality as long as the image is large enough and the size difference is quite substantial even if you were to use jpegs with more pixels.

**Comment by ebbit1q** [comment]
the jpeg version looks off, why are the corners purple? I think the png was better

**Comment by SlightlyCircuitous** [comment]
I am not sure why the jpeg corners are purple, you will have to ask GIMP. I agree with you that the png looks better anyway (it's brighter) so I have changed it back.

File: tokens.xml [hunk 1]
```diff
@@ -11593,7 +11593,7 @@ Throne of the Dead Three — Reveal the top ten cards of your library. Put a cre
         <card>
             <name>Kaito Shizuki Emblem</name>
             <text>Whenever a creature you control deals combat damage to a player, search your library for a blue or black creature card, put it onto the battlefield, then shuffle.</text>
-            <prop>
+            <prop>https://github.com/SlightlyCircuitous/image-storage/blob/main/karn_emblem_arena.png?raw=true
```
Comment (ebbit1q) [inline tokens.xml:11596]: what's this?

Comment (ebbit1q) [reply tokens.xml:11596]:
```suggestion
            <prop>
```

Comment (SlightlyCircuitous) [reply tokens.xml]: Not sure. must have accidentally added it pasted it there when I was adding the png link back.

**Review by ebbit1q — APPROVED** [review]
cool beans

**Pull request merged**